src/components/Dialog/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { createContext, type PropsWithChildren } from 'react';
+
+import DialogButton from '@components/Dialog/components/DialogButton';
+import DialogTitle from '@components/Dialog/components/DialogTitle';
+import * as styles from '@components/Dialog/style.css';
+
+interface DialogContextProps {
+  type: 'small' | 'medium';
+}
+
+interface DialogRootProps extends PropsWithChildren {
+  type: 'small' | 'medium';
+}
+
+export const DialogContext = createContext<DialogContextProps | null>(null);
+
+const DialogRoot = ({ children, type }: DialogRootProps) => {
```

Comment:
```suggestion

interface DialogContextProps {
  type: 'small' | 'medium';
}

interface DialogRootProps extends DialogContextProps {}

export const DialogContext = createContext<DialogContextProps | null>(null);

const DialogRoot = ({ children, type }: PropsWithChildren<DialogRootProps>) => {
```

이렇게 작성해볼 수 있을 것 같아요!

Reply:
[3e3ec86](https://github.com/YAPP-Github/23rd-Web-Team-2-FE/pull/39/commits/3e3ec8688cce1354d7875ecf6daefbbb1e5fdd75)
감사합니다! 여기에서 반영해보았습니다 ㅎㅎ